### PR TITLE
docs: updated the broken link and highlighted few text

### PR DESCRIPTION
**Pull request by AnkitKumarvaid**
updates:
1-fixed: broken link for downloading node
2-highlighted the keyword is required in 3 sections

### CODE REVIEW

**Comment by AnkitKumarvaid** [comment]
@re-Tick @nehagup I have closed the previous pr and have created a new one for the same readme update issue. please review this PR.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This is the client SDK for the [Keploy](https://github.com/keploy/keploy) testin
 8. [Community support](#community-support)
 
 ## Installation
-1. First install [Node.js](http://nodejs.org/). Then,
+1. First install [Node.js](https://nodejs.org/en/download). Then,
 ```bash
 # for npm package manager
 npm i https://github.com/keploy/typescript-sdk
```
Comment (re-Tick) [inline README.md:22]: Please seperate the block for npm and yarn.

**Comment by AnkitKumarvaid** [comment]
@re-Tick plz check, I have updated the readme according to your explanation.

**Comment by re-Tick** [comment]
LGTM😀

**Pull request merged**